Tests/EndpointTests/UnsignedInt128Tests.swift

```diff
@@ -0,0 +1,401 @@
+    // }
+
+    // @available(swiftEndpointApplePlatforms 15, *)
+    // @Test(arguments: generateRandomUInt128Pairs())
+    // func `test %= against UInt128`(lhs: UInt128, rhs: UInt128) {
+    //     var uint128 = lhs
+    //     uint128 %= rhs
+    //     var unsignedInt128 = UnsignedInt128(lhs)
+    //     unsignedInt128 %= UnsignedInt128(rhs)
+    //     #expect(uint128 == unsignedInt128)
+    // }
+
+    // @available(swiftEndpointApplePlatforms 15, *)
+    // @Test(arguments: generateRandomUInt128Pairs())
+    // func `test * against UInt128`(lhs: UInt128, rhs: UInt128) {
+    //     let uint128 = lhs % rhs
+    //     let unsignedInt128 = UnsignedInt128(lhs) % UnsignedInt128(rhs)
+    //     #expect(uint128 == unsignedInt128)
+    // }
+
+    // @available(swiftEndpointApplePlatforms 15, *)
+    // @Test(arguments: generateRandomUInt128Pairs())
+    // func `test &= against UInt128`(lhs: UInt128, rhs: UInt128) {
+    //     let uint128 = lhs % rhs
+    //     let unsignedInt128 = UnsignedInt128(lhs) % UnsignedInt128(rhs)
+    //     #expect(uint128 == unsignedInt128)
+    // }
+
+    // @available(swiftEndpointApplePlatforms 15, *)
+    // @Test(arguments: generateRandomUInt128Pairs())
+    // func `test |= against UInt128`(lhs: UInt128, rhs: UInt128) {
+    //     var uint128 = lhs
+    //     uint128 |= rhs
+    //     var unsignedInt128 = UnsignedInt128(lhs)
+    //     unsignedInt128 |= UnsignedInt128(rhs)
+    //     #expect(uint128 == unsignedInt128)
+    // }
+
+    // @available(swiftEndpointApplePlatforms 15, *)
+    // @Test(arguments: generateRandomUInt128Pairs())
+    // func `test ^= against UInt128`(lhs: UInt128, rhs: UInt128) {
+    //     var uint128 = lhs
+    //     uint128 ^= rhs
+    //     var unsignedInt128 = UnsignedInt128(lhs)
+    //     unsignedInt128 ^= UnsignedInt128(rhs)
+    //     #expect(uint128 == unsignedInt128)
+    // }
+
+    // @available(swiftEndpointApplePlatforms 15, *)
+    // @Test(arguments: generateRandomUInt128Pairs())
+    // func `test magnitude against UInt128`(lhs: UInt128, _: UInt128) {
+    //     let uint128 = lhs.magnitude
+    //     let unsignedInt128 = UnsignedInt128(lhs).magnitude
+    //     #expect(uint128 == unsignedInt128)
+    // }
+
+    @available(swiftEndpointApplePlatforms 15, *)
+    @Test func `verify init(_ description: String) against UInt128`() throws {
+        for (lhs, _) in generateRandomUInt128Pairs() {
+            let uint128 = try #require(UInt128(lhs.description))
+            let unsignedInt128 = try #require(UnsignedInt128(lhs.description))
+            #expect(uint128 == unsignedInt128)
+            #expect(uint128 == lhs)
+        }
+    }
+
+    @available(swiftEndpointApplePlatforms 15, *)
+    private func generateRandomUInt128Pairs() -> some Sequence<(UInt128, UInt128)> {
+        (0..<100_000).map { _ in
+            (UInt128.random(in: .min ... .max), UInt128.random(in: .min ... .max))
+        }[0...10]
```

Comment:
Need to remove the `[0...10]` before merging:
```suggestion
        }
```